multi-agent/lib/mas/elements/nodes/a2a_agent/a2a_agent_node.py

```
@@ -234,6 +228,23 @@ def _build_agent_results_context(self, thread_id: str) -> Optional[ChatMessage]:
+        over HTTP), so we bridge into async via the same utility the
+        A2AProvider uses for send_message_sync.
+        """
+        if not self._auth_credential:
+            return
+        try:
+            with get_async_bridge() as bridge:
+                headers = bridge.run(self._auth_credential.get_headers())
+            if headers:
+                self.a2a_provider.update_headers(headers)
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)